assignments/objects.js

```diff
@@ -18,28 +18,76 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi Zu",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+
+const kennan = {
+  "id": 2,
+  "name": "Kennan Art",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const keven = {
+  "id": 3,
+  "name": "Keven Mummery",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const gannie = {
+  "id": 4,
+  "name": "Gannie Wannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const antonietta = {
+  "id": 5,
+  "name": "Antonietta Sorensen",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+// console.log(mitzi.name);
 
 // Kennan's ID
+// console.log(kennan.id);
+
 
 // Keven's email
+// console.log(keven.email);
 
 // Gannie's name
+// console.log(gannie.name);
 
 // Antonietta's Gender
+// console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
-// Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
+kennan.speak = "Hello, my name is Kennan!"
```

Comment:
```suggestion
kennan.speak = () => "Hello, my name is Kennan!"
```
I'd have rather seen the speak method added in the original object, but you can do it this way. However, the way you have it here, you gave Kennan the property speak, which is a string of "Hello, my name is Kennan!".  This is why it is working when you call kennan.speak . To call a method we would want to call kennan.speak() , and we need to make speak a method.